scientific/spectroscopy/specpr.ksy

```diff
@@ -0,0 +1,254 @@
+meta:
+  id: specpr
+  file-extension: spec
+  endian: be
+  license: unlicense
+  #encoding: "utf-8"
+  encoding: "ascii"
+  title: "SPECtrum Processing Routines Data Format 3/4/88"
+doc: >
+  Specpr records are fixed format, 1536 bytes/record. Record number counting starts at 0. Binary data are in IEEE format real numbers and non-byte swapped integers (compatiible with all Sun Microsystems, and Hewlett Packard workstations (Intel and some DEC machines are byte swapped relative to Suns and HPs). Each record may contain different information according to the following scheme.
+  You can get some library of spectra from ftp://ftpext.cr.usgs.gov/pub/cr/co/denver/speclab/pub/spectral.library/splib06.library/

```

Comment:
Generally, that's not what supposed to be `doc` element for top-level type. Top-level docstring is supposed to explain *what* is this format is all about, i.e. what are the contents, where / how / what for it is used, who designed it, who are the providers of the software, etc. Implementation details are supposed to be put into individual types documentation.

Last, but not least: I suppose in this case it's ok to just copy text descriptions verbatim, as this is probably public domain work (being US government official's work), but in general case you can't just copy text as is, especially not under different license — it would be copyright violation.